lib/rules/require-valid-alt-text.js

```diff
@@ -76,20 +66,6 @@ export default class RequireValidAltText extends Rule {
             }
           }
 
-          // if the role value is a mustache statement we can not validate it
-          if (hasAltAttribute && hasRole && !roleValue.type) {
-            if (
-              ['none', 'presentation'].includes(roleValue.trim().toLowerCase()) &&
-              altValue !== ''
-            ) {
-              this.logNode({
-                message:
-                  'The `alt` attribute should be empty if `<img>` has `role` of `none` or `presentation`',
-                node,
-              });
-            }
-          }
```

Comment:
I don't think this should be removed. It is not valid to have a non-empty alt attribute if `role="presentation"` or `role="none"`